src/Raven.Server/Documents/Handlers/CountersHandler.cs

```diff
@@ -754,6 +759,119 @@ private static void ThrowMissingDocument(string docId)
         {
             throw new DocumentDoesNotExistException(docId, "Cannot operate on counters of a missing document.");
         }
+
+        public class ExecuteFixCounterGroupsCommand : TransactionOperationsMerger.MergedTransactionCommand
+        {
+            private readonly string _docId;
+            private readonly DocumentDatabase _database;
+
+            public ExecuteFixCounterGroupsCommand(DocumentDatabase database, string docId)
+            {
+                _docId = docId;
+                _database = database;
+            }
+
+            protected override long ExecuteCmd(DocumentsOperationContext context)
+            {
+                var numOfCounterGroupFixed = _database.DocumentsStorage.CountersStorage.FixCountersForDocument(context, _docId);
+                return numOfCounterGroupFixed;
+            }
+
+            public override TransactionOperationsMerger.IReplayableCommandDto<TransactionOperationsMerger.MergedTransactionCommand> ToDto<TTransaction>(TransactionOperationContext<TTransaction> context)
+            {
+                throw new NotImplementedException();
+            }
+        }
+
+        [RavenAction("/databases/*/counters/fix", "PATCH", AuthorizationStatus.ValidUser, EndpointType.Write)]
```

Comment:
would go with `GET` here, so it will be easier for users to execute this EP
maybe `/databases/*/counters/fix/ravendb-22835`
@ppekrol thoughts?